articles/libraries/lock/v9/index.md

```diff
@@ -76,6 +75,28 @@ If you are using browserify to build your project, you will need to add the foll
 }
 ```
 
+## Initialization
+
+Auth0Lock v9 can be initialized with just a clientID and domain, or it can also take a third parameter, an `options` object.
+
+```js
+// Initialize with clientID and domain
+var lock = new Auth0Lock(clientID, domain);
+
+// Or, initialize with options
+var lock = new Auth0Lock(clientID, domain, options);
+```
+
+### Parameters
+
+The possible parameters are detailed below.
+
+* **clientID {String}**: Your application clientID in Auth0.
+* **domain {String}**: Your Auth0 domain. Usually ```<account>.auth0.com```.
+* **options {Object}**:
+  * _**cdn {String}**_: Use as CDN base url. Defaults to `domain` if it doesn't equal `*.auth0.com`.
```

Comment:
This doesn't look good, keep only the bold markdown ie `_**cdn {String}**_` -> `**cdn {String}**`